tool/tctl/common/auth_command.go

```diff
@@ -124,6 +125,7 @@ func (a *AuthCommand) Initialize(app *kingpin.Application, config *service.Confi
 	a.authSign.Flag("db-service", `Database to generate identity file for. Mutually exclusive with "--app-name".`).StringVar(&a.dbService)
 	a.authSign.Flag("db-user", `Database user placed on the identity file. Only used when "--db-service" is set.`).StringVar(&a.dbUser)
 	a.authSign.Flag("db-name", `Database name placed on the identity file. Only used when "--db-service" is set.`).StringVar(&a.dbName)
+	a.authSign.Flag("jks-password", `JKS password used for cassandra format for Keystore and Truststore protection.`).StringVar(&a.jksPassword)
```

Comment:
@smallinsky Can we remove this flag and always generate the password? This will make it more secure as then we can force the strong password every time.

Reply:
removed.